projects/kit/src/lib/utils/date/tests/get-date-segment-value-length.spec.ts

```diff
@@ -0,0 +1,15 @@
+import {getDateSegmentValueLength} from '../date-segment-value-length';
+
+describe('getDateSegmentValueLength', () => {
+    it('short date', () => {
+        expect(getDateSegmentValueLength('mm.yy')).toEqual({day: 2, month: 2, year: 2});
```

Comment:
`{day: 2, month: 2, year: 2}` is expected behaviour ?

not `{day: 0, month: 2, year: 2}` ?